providers/shared/entrances/occurrences/id.ftl

```diff
@@ -0,0 +1,18 @@
+[#ftl]
+
+[@addEntrance
+    type=OCCURRENCES_ENTRANCE_TYPE
+    properties=
+        [
+            {
+                "Type"  : "Description",
+                "Value" : "Provides the state of all occurrences within a district"
```

Comment:
It sounds like the use-case for this is primarily internal to Hamlet Deploy. 

The naming pattern we use for internal macros is not available to us here because of the dynamic macro evaluation but perhaps the Description here should reflect this?

Reply:
I think the macros you might be referring too are actually internal to functionality within the engine not necessarily to hamlet deploy or even the engine itself. 

The occurrence structure is a fairly standard interface now so I think it would be safe to call its structure a stable public interface that people can use outside of hamlet if they want.

This isn't really that different to the blueprint entrance which includes the occurrence information anyway